Makefile

```diff
@@ -28,7 +29,9 @@ common/$(PN):
 install-common:
 	$(INSTALL_DIR) "$(DESTDIR)$(UDEVDIR)"
 	$(INSTALL_DIR) "$(DESTDIR)$(ENVDIR)"
+	$(INSTALL_DIR) "$(DESTDIR)$(POLKDIR)"
```

Comment:
The polkit dir and file is only needed for arm not x86_64.  Please move these two lines under that conditional.